python/tests/test_base_install/test_graphql/test_rolling_expanding.py

```diff
@@ -2041,3 +2041,242 @@ def test_wrong_window():
     """
     queries_and_exceptions.append((query, mismatch_exception))
     run_group_graphql_error_test(queries_and_exceptions, graph)
+
+def test_alignment():
+    g = Graph()
+    dt1 = datetime(2025, 3, 15, 14, 37, 52)     # March 15
+    dt2 = datetime(2025, 7, 8, 9, 12, 5)        # July 8
+    dt3 = datetime(2025, 11, 22, 21, 45, 30)    # November 22
+
+    g.add_node(dt1, 1)
+    g.add_node(dt2, 1)
+    g.add_node(dt3, 1)
+
+    query = """
+    {
```

Comment:
Can you just chuck in a rolling/expanding test for Node, nodes, edge, edges, pathFromNode, PathFromGraph. I know its a little extra, but just want to make sure we have them called at least once